src/elements/hx-tab/index.spec.js

```diff
@@ -0,0 +1,85 @@
+import { fixture, oneEvent, expect } from '@open-wc/testing';
+
+/**
+ * <hx-tab> component tests
+ *
+ * @type HXTabElement
+ *
+ */
+describe('<hx-tab> component tests', () => {
```

Comment:
### Your tests should fail based on this error, otherwise it's not validating what we need to resolve Issue #516.

<img width="1266" alt="Screen Shot 2020-06-17 at 7 52 54 PM" src="https://user-images.githubusercontent.com/10120600/84965394-6c9ceb80-b0d4-11ea-8646-d98f1b8a8140.png">

Reply:
@BY00565233, I was able to replicate the above failures with our component testing pipeline:

<img width="959" alt="Screen Shot 2020-07-28 at 5 48 17 PM" src="https://user-images.githubusercontent.com/10120600/88729936-95b09300-d0fa-11ea-9d51-4e0db54324ce.png">

Please add the skipped tests referenced in PR #723.  Once the dynamic tab bug patch is implemented, we will include these tests to our test run.